apps/docs/components/.vuepress/components/Showcase.vue

```diff
@@ -67,21 +68,44 @@ export default {
   }),
   mounted() {
     document.addEventListener('mousemove', this.mouseMoveListener);
+    document.addEventListener('touchmove', this.touchMoveListener);
     document.addEventListener('mouseup', this.mouseUpListener);
+    document.addEventListener('touchup', this.touchUpListener)
   },
   unmounted() {
     document.removeEventListener('mousemove', this.mouseMoveListener);
+    document.removeEventListener('touchmove', this.touchMoveListener);
     document.removeEventListener('mouseup', this.mouseUpListener);
+    document.removeEventListener('touchUp', this.touchUpListener);
   },
   methods: {
     mouseDownListener(e) {
       if (e.target === this.$refs.handlerRef) {
         this.isHandlerDragging = true;
       }
     },
+    touchDownListener(e) {
+      if (e.target === this.$refs.handlerRef) {
+        this.isHandlerDragging = true;
+      }
+    },
+    touchUpListener(e) {
+      this.isHandlerDragging = false;
+    },
     mouseUpListener() {
       this.isHandlerDragging = false;
     },
+    touchMoveListener(e){
+      if (!this.isHandlerDragging) return false;
+      
+      const containerOffsetLeft = this.$refs.wrapperRef.getBoundingClientRect().left;
+      const pointerRelativeXpos = e.touches[0].pageX - containerOffsetLeft;
+      const minWidth = 320;
+
+      this.$refs.previewElementRef.style.maxWidth = `${Math.max(minWidth, pointerRelativeXpos)}px`;
+      this.$refs.previewElementRef.style.maxWidth = `${Math.max(minWidth, pointerRelativeXpos)}px`;
+
+    },
```

Comment:
Can we please do one method for both? also there is 2x `previewElementRef.style.maxWidth `
```suggestion
    moveListener(e){
      if (!this.isHandlerDragging) return false;
      
      const containerOffsetLeft = this.$refs.wrapperRef.getBoundingClientRect().left;
      const pointerRelativeXpos = ((e?.touches[0] && e?.touches[0]?.pageX) || e?.clientX) - containerOffsetLeft;
      const minWidth = 320;

      this.$refs.previewElementRef.style.maxWidth = `${Math.max(minWidth, pointerRelativeXpos)}px`;
    },
```